manual/manual/refman/exten.etex

```diff
@@ -460,40 +460,40 @@ suspend the evaluation of @expr@ as a regular abstraction would.  The
 syntax has been chosen to fit nicely in the context of function
 declarations, where it is generally used. It is possible to freely mix
 regular function parameters with pseudo type parameters, as in:
-\begin{verbatim}
-        let f = fun (type t) (foo : t list) -> ...
-\end{verbatim}
+\begin{caml_example*}{verbatim}
+        let f = fun (type t) (foo : t list) -> assert false[@ellipsis]
+\end{caml_example*}
 and even use the alternative syntax for declaring functions:
-\begin{verbatim}
-        let f (type t) (foo : t list) = ...
-\end{verbatim}
+\begin{caml_example*}{verbatim}
+        let f (type t) (foo : t list) = assert false[@ellipsis]
+\end{caml_example*}
 If several locally abstract types need to be introduced, it is possible to use
 the syntax
 @"fun" '(' "type" typeconstr-name_1 \ldots typeconstr-name_n ')' "->" expr@
 as syntactic sugar for @"fun" '(' "type" typeconstr-name_1 ')' "->" \ldots "->"
 "fun" '(' "type" typeconstr-name_n ')' "->" expr@. For instance,
-\begin{verbatim}
-        let f = fun (type t u v) -> fun (foo : (t * u * v) list) -> ...
-        let f' (type t u v) (foo : (t * u * v) list) = ...
-\end{verbatim}
+\begin{caml_example*}{verbatim}
+        let f = fun (type t u v) -> fun (foo : (t * u * v) list) -> assert false[@ellipsis]
+        let f' (type t u v) (foo : (t * u * v) list) = assert false[@ellipsis]
+\end{caml_example}
 
 This construction is useful because the type constructors it introduces
 can be used in places where a type variable is not allowed. For
 instance, one can use it to define an exception in a local module
 within a polymorphic function.
-\begin{verbatim}
+\begin{caml_example*}{verbatim}
         let f (type t) () =
           let module M = struct exception E of t end in
           (fun x -> M.E x), (function M.E x -> Some x | _ -> None)
-\end{verbatim}
+\end{caml_example*}
```

Comment:
This environment change and the one below look like they are likely to conflict with #1209 for no very good reason.

Reply:
True, I will remove them. (I could also remove all changes to `exten.etex` since they are more here as an example than anything else.).